internal/builders/generic/README.md

```diff
@@ -73,6 +109,33 @@ jobs:
     uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@main
     with:
       base64-subjects: "${{ needs.build.outputs.digest }}"
+
```

Comment:
same with pinning main in this action above

Reply:
Done.